eo-runtime/src/main/eo/org/eolang/collections/list.eo

```diff
@@ -95,7 +95,25 @@
         &.f x > @
 
   # For each array element dataize the object
-  [f] > each /bool
+  # Here "f" must be an abstract object with
+  # one free attribute, the element of the
+  # array.
+  [f] > each
+    [f] > iterate
+      seq > @
+        memory 0 > idx
+        idx.write 0
```

Comment:
@l3r8yJ I think there is no need to do this since `memory` has been already initialized by `0`.